conf/base.config

```diff
@@ -10,11 +10,12 @@
 
 process {
 
+    // TODO nf-core: Check the defaults for all processes
     cpus   = { check_max( 1    * task.attempt, 'cpus'   ) }
     memory = { check_max( 6.GB * task.attempt, 'memory' ) }
     time   = { check_max( 4.h  * task.attempt, 'time'   ) }
 
-    errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'finish' }
+    errorStrategy = { task.exitStatus in ((130..145) + 104) ? 'retry' : 'finish' }
     maxRetries    = 1
```

Comment:
You might want to up this number as it is no longer controlled by the main `sanger.config`

Reply:
Sorry, I don't quite get this comment. This is the list of error codes on which the pipelines should be retried. Do you mean to top it up with some other error codes?

Reply:
Figured it out. Asked Matthieu.